Additions/ISum.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Additions
+{
+    public interface ISum
+    {
+        public dynamic Add(dynamic a, dynamic b);
+    }
```

Comment:
You have no way to parse a single variable sch as an array. Please include that here